lib/email_api/emailer.py

```diff
@@ -16,6 +16,8 @@
 from structs.email.smtp_account import SMTPAccount
 from structs.email.email_template_details import EmailTemplateDetails
 
+import css_inline
```

Comment:
As far as I can see, you only use one method from `css_inline`:
```suggestion
from css_inline import CSSInliner
```